app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+  
+  validates :name,        presence: true
+  validates :email,       presence: true, format: { with:VALID_EMAIL_REGEX }
```

Comment:
This is cool and I see if working on page.

Reply:
I can sign up with the same email address multiple times. Should this be the case or should they be unique?

Reply:
I understand email address don't need to be unique.